modules/apps/foundation/frontend-js/frontend-js-spa-web/src/main/java/com/liferay/frontend/js/spa/web/servlet/taglib/util/SPAUtil.java

```diff
@@ -81,6 +82,16 @@ public String getExcludedPaths() {
 		return jsonArray.toString();
 	}
 
+	public String getLoginRedirect(HttpServletRequest request) {
+		String loginPortletNamespace = PortalUtil.getPortletNamespace(
+			PropsUtil.get(PropsKeys.AUTH_LOGIN_PORTLET_NAME));
+
+		String loginRedirectParameter = loginPortletNamespace + "redirect";
+
+		return GetterUtil.getString(
```

Comment:
@brianchandotcom To use ParamUtil here I would have to add a dependency to "provided group: "javax.portlet", name: "portlet-api", version: "2.0""

If you want I can add it. I didn't think it was worth adding a dependency though.